saleor/graphql/payment/mutations/transaction/payment_gateway_initialize.py

```diff
@@ -65,7 +65,10 @@ class Meta:
             "Initializes a payment gateway session. It triggers the webhook "
             "`PAYMENT_GATEWAY_INITIALIZE_SESSION`, to the requested `paymentGateways`. "
             "If `paymentGateways` is not provided, the webhook will be send to all "
-            "subscribed payment gateways." + ADDED_IN_313 + PREVIEW_FEATURE
+            "subscribed payment gateways. There is a limit of transaction items per"
+            "checkout / order, configurable in settings by TRANSACTION_ITEMS_LIMIT."
```

Comment:
can't you use `f"{settings.TRANSACTION_ITEMS_LIMIT}"` here ? I don't think that writing that this value is configurable from settings will give frontend developer any hint.